pkg/scheduler/plugins/drf/drf.go

```diff
@@ -66,6 +70,8 @@ func (drf *drfPlugin) OnSessionOpen(ssn *framework.Session) {
 		drf.totalResource.Add(n.Allocatable)
 	}
 
+	namespaceOrderEnabled := ssn.NamespaceOrderEnabledInPlugin(drf.Name())
```

Comment:
is there anyway to remove `namespaceOrderEnabled`? That's error prone :)

Reply:
I refactor the code in allocate, there is no `namespaceOrderEnabled` in allocation func now.

And, `namespaceOrderEnabled` in `drf` plugin is also refactored to be more readable.